src/MongoDB.Driver/FilterDefinitionBuilder.cs

```diff
@@ -1493,6 +1493,16 @@ public FilterDefinition<TDocument> Where(Expression<Func<TDocument, bool>> expre
 
     internal sealed class AndFilterDefinition<TDocument> : FilterDefinition<TDocument>
     {
+        #region static
+        private static readonly string[] __filtersThatAllowOnlyDollarForm = new[]
+        {
+            "$geoWithin",
```

Comment:
the first two are mentioned in the ticket. I also checked the 3rd and 4th and they have the same behavior. As I understand there are no more geo query selectors (based on this https://docs.mongodb.com/manual/reference/operator/query-geospatial/)

Reply:
Please rename this variable to:

    __operatorsThatCantBeCombined

Reply:
Nit: Should this be `__operatorsThatCannotBeCombined`? We typically seem to spell out contractions.

Reply:
done